dmutils/content_loader.py

```diff
@@ -319,6 +318,10 @@ def get_question_by_slug(self, question_slug):
             if question.get('slug') == question_slug:
                 return question
 
+    def inject_brief_questions_into_boolean_list_question(self, brief):
```

Comment:
Yeah, I tend to agree with your final comment.  We only need these for briefs, so with our really specific method naming it's obvious what they're doing.